sourcegraph/jetbrains
# Add support for local symf context

Fixes https://github.com/sourcegraph/jetbrains/issues/259

## Test plan

Manual testing using full QA guide.

Other than that:
1) Enable context
2) Ask repo-specific question
3) Check if response is based on the understanding of the local context

4) Disable context
5) Ask repo-specific question
6) Check if response is lacking previous insight

7) Click on refresh button and check if reindexing is in progress
8) Click on help button and check if it opens proper documentation page
